do.sh

```diff
@@ -317,9 +347,32 @@ function mine_data() {
                             | grep ovn-scale | head -3)
     python3 ${topdir}/utils/process-stats.py \
         resource-usage-report-worker.html ${resource_usage_logs}
+    deactivate
+
     popd
 }
 
+function get_tester_ip() {
+    local test_file=$1
+
+    node_net=$(${ovn_fmn_get} ${test_file} cluster node_net --default=192.16.0.0/16)
+    node_cidr=${node_net#*/}
+    node_ip=${node_net%/*}
+    ip_index=1
+    clustered_db=$(${ovn_fmn_get} ${test_file} cluster clustered_db --default=True)
+    # The yaml loader normalizes "yes" and "no" values for clustered_db into
+    # "True" and "False"
+    if [ "${clustered_db}" = "True" ]; then
+        (( ip_index += 3 ))
+    else
+        (( ip_index += 1 ))
+    fi
+    n_relays=$(${ovn_fmn_get} ${test_file} cluster n_relays --default=0)
+    (( ip_index += ${n_relays} ))
+    tester_ip=$(${ovn_fmn_ip} ${node_net} ${node_ip} ${ip_index})
+    echo "${tester_ip}/${node_cidr}"
+}
+
```

Comment:
Can we make ovn-tester use the first IP and make the ovn-tester.py figure out IPs for the rest of the deployments?
Specifying defaults in two different places seems very error-prone.

Reply:
I'll need to have another look at how ovn-fake-multinode's IP address assignment works. When doing this work, I did not think it would be possible because of inflexibilities in how ovn-fake-multinode assigns central and relay addresses. However, I may be able to cheat it by just bumping the starting address by 1 when starting central and relay containers in ovn-tester.